leasing/models/types.py

```diff
@@ -24,6 +24,7 @@
 
 
 class PayableRent(TypedDict):
+    # FIXME name clash with existing class PayableRent in rent.py
```

Comment:
Perhaps fixable in this PR?

Reply:
Technically yes, but I skipped thinking about it because customer wants this feature out ASAP. Suggestions for replacement name?